code_samples/back_office/dashboard/article_tab/config/custom_services.yaml

```
@@ -10,4 +10,4 @@ services:
         arguments:
             $groupIdentifier: 'custom_group'
         tags:
-            - { name: ibexa.admin_ui.component, group: 'dashboard-blocks' }
+            - { name: ibexa.twig.component, group: 'dashboard-blocks' }
```

Shouldn't the group be `admin-ui-dashboard-blocks` according to [Back office > Back office elements > Custom components > Admin UI](https://ez-systems-developer-documentation--2739.com.readthedocs.build/en/2739/administration/back_office/back_office_elements/custom_components/#admin-ui)?

```suggestion
            - { name: ibexa.twig.component, group: 'admin-ui-dashboard-blocks' }
```